code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -886,6 +887,12 @@
 	obj_flags |= EMAGGED
 	locked = FALSE
 
+/obj/machinery/turretid/attack_robot(mob/user)
+	if(!ailock || IsAdminGhost(user))
```

Comment:
What about syndicate borgs?

Reply:
No silicon control means no silicon control.

Reply:
Syndicate borgs can also have their laws changed to assist the station and stop the ops while still having the syndicate faction. I might even make the panel Syndicate leader access only.

Reply:
Most leaders are inept and die early so please leave all nukies able to use it.